pygsp/graphs/fourier.py

```diff
@@ -171,7 +171,7 @@ def compute_fourier_basis(self, n_eigenvectors=None):
 
         # TODO: handle non-symmetric Laplacians. Test lap_type?
         if n_eigenvectors == self.n_vertices:
-            self._e, self._U = np.linalg.eigh(self.L.toarray())
+            self._e, self._U = linalg.eigh(self.L.toarray(order='F'), overwrite_a=True)
```

Comment:
order='F' as lapack is fortran ordered, and re-use space for eigenvectors.

Reply:
Thanks! Does the Fortran order make the EVD faster?

Reply:
It should avoid the matrix being transposed from C order to F order when it is pushed out to lapack.

Reply:
Makes sense, thanks. At some point we should do a proper benchmark and find the best settings for the partial and full EVDs.